openproblems/tasks/cell_cell_communication/api.py

```diff
@@ -0,0 +1,90 @@
+from ...data.sample import load_sample_data
+from ...tools.decorators import dataset
+
+import numpy as np
+import pandas as pd
+import scanpy as sc
+
+
+def check_dataset(adata):
+    """Check that dataset output fits expected API."""
+    assert "label" in adata.obs
+    assert "ccc_target" in adata.uns
+    assert "response" in adata.uns["ccc_target"]
+    assert np.issubdtype(adata.uns["ccc_target"]["response"].dtype, int)
```

Comment:
Can we also add `assert np.all(np.isin(adata.uns["ccc_target"]["response"], [0,1]))`

Reply:
Done.